pkg/catalog/types.go

```diff
@@ -25,17 +26,18 @@ type CatalogConnection interface {
 // ChartMeta represents a single helm chart
 // NOTE: This structure is derived from the JSON return from the catalog service /charts/ endpoint
 type ChartMeta struct {
-	Name        string             `yaml:"name"`
-	Version     string             `yaml:"version"`
-	Description string             `yaml:"description"`
-	ApiVersion  string             `yaml:"apiVersion"`
-	AppVersion  string             `yaml:"appVersion"`
-	KubeVersion string             `yaml:"kubeVersion"`
-	Type        string             `yaml:"type"`
-	Urls        []string           `yaml:"urls"`
-	Created     time.Time          `yaml:"created"`
-	Digest      string             `yaml:"digest"`
-	Annotations map[string]string  `yaml:"annotations"`
+	Name        string            `yaml:"name"`
+	Version     string            `yaml:"version"`
+	Description string            `yaml:"description"`
+	ApiVersion  string            `yaml:"apiVersion"`
+	AppVersion  string            `yaml:"appVersion"`
+	KubeVersion string            `yaml:"kubeVersion"`
+	Type        string            `yaml:"type"`
+	Urls        []string          `yaml:"urls"`
+	Created     time.Time         `yaml:"created"`
+	Digest      string            `yaml:"digest"`
+	Annotations map[string]string `yaml:"annotations"`
+	Deprecated  string            `yaml:"deprecated"`
```

Comment:
Why is the data type not `bool` instead? Refer https://github.com/helm/helm/blob/1b21f04b3b6960fd48bfee99ae2cfb37c0be708a/pkg/chart/v2/metadata.go#L74